apps/docs/docusaurus.config.ts

```diff
@@ -0,0 +1,130 @@
+import { themes as prismThemes } from "prism-react-renderer"
+import type { Config } from "@docusaurus/types"
+import type * as Preset from "@docusaurus/preset-classic"
+
+const config: Config = {
+    title: "Bandada docs",
+    tagline: "Bandada documentation",
+    favicon: "img/favicon.ico",
+
+    // Set the production url of your site here
+    url: "https://your-docusaurus-site.example.com",
```

Comment:
We should change also these default values `url`, `organizationName`, `projectName` and `editUrl`. Otherwise, if aren't needed, we can get rid of them for now.

Reply:
I will update these values, thanks.

Reply:
I updated everything with the new Bandada org because we will move the repo this week.